server.js

```diff
@@ -1,6 +1,8 @@
 // server.js
-const jsonServer = require('json-server')
-const bodyParser = require('body-parser')
+// const jsonServer = require('json-server')
```

Comment:
can these be removed?

Reply:
Removed the comments and also fixed the eslint warnings.